boards/opentitan/earlgrey-cw310/Makefile

```diff
@@ -6,7 +6,8 @@ PLATFORM=earlgrey-cw310
 FLASHID=--dev-id="0403:6010"
 RISC_PREFIX ?= riscv64-elf
 QEMU ?= ../../../tools/qemu/build/qemu-system-riscv32
-
+# Override for the entry point
+QEMU_ENTRY_POINT=0x20000400
```

Comment:
How do we know what value to set this to?

Reply:
It's the start address of Tock. It should only change if we change the linker

Reply:
> How do we know what value to set this to?

It's basically the `ORIGIN(rom)` + [`end_manifest`](https://github.com/tock/tock/blob/master/boards/opentitan/earlgrey-cw310/layout.ld#L41).

Reply:
Can we write this down in the Makefile?

Reply:
Added this as a comment to the Makefile